pkgs/development/libraries/science/math/openblas/default.nix

```diff
@@ -116,6 +116,15 @@ stdenv.mkDerivation rec {
     sha256 = "14jxh0v3jfbw4mfjx4mcz4dd51lyq7pqvh9k8dg94539ypzjr2lj";
   };
 
+  # remove this when updating from 0.3.13
```

Comment:
```suggestion
  # apply https://github.com/xianyi/OpenBLAS/pull/3060 to fix a crash on arm
  # remove this when updating from 0.3.14
```

Reply:
Thanks for the suggestions. I agree with this except for this small detail:

You changed the version number in the comment to `0.3.14`

`0.3.13` is the release we have currently.

`0.3.14` was released a a few days ago. It would most likely already have a fix for this, but there is an important unrelated regression in that release. So we probably shouldn't take that regressed release now, but either merge this or wait for a new release that also fixes the regression.

Reply:
I have changed the comment so that it's accurate again in https://github.com/NixOS/nixpkgs/pull/117004/commits/7d472d8ecafa28851e142e8ceef8b65a48b35c9d.

Reply:
Yes, thanks, I meant to say when updating *to* 0.3.14.